### ISSUE TITLE
YouCompleteMe support

### ISSUE THREAD

**Issue by collegeimprovements** [Shougo/neosnippet.vim#277]
Hi Shougo,
Thanks for writing this awesome plugin.
Is there any way by which we can see the snippets in YouCompleteMe completion list.
This is certainly not a bug. But it would be awesome if that is possible.
And by looking at [this](https://www.youtube.com/watch?v=1KjR70JNVmw)  video I'm sure it's possible.
I just don't know how.
Please help if possible. 

Thanks a lot for your all open-source contributions :)

**Comment by Shougo** [comment]
You can get snippet candidates by `values(neosnippet#helpers#get_snippets())`.

**Comment by collegeimprovements** [comment]
Thanks :) :+1:

**Comment by juanes852** [comment]
Hi,
I try to use NeoSnippet with YouCompleteMe, but It doesn't work: snippets don't appear in popup menu. I've tried to write in my ~/.vimrc « values(neosnippet#helpers#get_snippets()) ». I've also tried download https://gist.github.com/sgur/5807738 in ~/.vim/bundle/YouCompleteMe/third_party/ycmd/ycmd/completers/general without any result, and I've also changed others files without any result. I don't uderstand, could you help me please ? I wan't use YCM in Neovim, because there are any bugs in Deoplete.

**Comment by Shougo** [comment]
I don't use neosnippet with YCM....

>  I wan't use YCM in Neovim, because there are any bugs in Deoplete.

Any bugs?  Interesting.
Please create new issue in deoplete.

**Comment by majidebr** [comment]
Hi @Shougo,
I'd like to use neosnippet with YCM, without neocomplete.
Could you please let me know how to use `values(neosnippet#helpers#get_snippets())` as you suggesed? I tried by copying this to my .vimrc file, but it didn't help.

**Comment by Shougo** [comment]
I think you don't load neosnippet.
You can check the result by `:echo values(neosnippet#helpers#get_snippets())` after loading.

**Comment by majidebr** [comment]
Thanks for the quick reply @Shougo .
I checked this and it told me that the snippets are not installed. I installed neo-snippets and now it shows me the snippetes for my file type(tex). But still YCM does not show the snippets in the suggestions.
I believe my problem is with what I put in my .vimrc. I have copied `values(neosnippet#helpers#get_snippets())` in my .vimrc and I get this error when launching vim:
`E492: Not an editor command: values(neosnippet#helpers#get_snippets())`
